src/legacy/core_plugins/kbn_vislib_vis_types/public/editors/point_series.tsx

```diff
@@ -77,9 +79,61 @@ function PointSeriesOptions(props: VisOptionsProps<BasicVislibParams>) {
             }
           />
         )}
+
+        <SwitchOption
```

Comment:
There are a couple of TypeScript issues in this file that need to be resolved. Also I think Threshold lines are an awesome new feature, that it deserves it's own panel. So maybe we could move all those new setting into its own `EuiPanel` below the `GridOptions` in this file?